scania-digital-design-system/sdds
# fix(tegel): remove base value and use px instead of rem

**Describe pull-request**  
Refactored tegel to not import reboot.scss. This removes the font-size: 4px on on the html tags. Removed the use of rem is all components/stories. Refactored all use of rem to px in theme/core. Refactored all use of rem to px in spacing and typography. 


**Solving issue**  
Fixes: [AB#2522](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/2522)

**How to test**  
1. Checkout branch and run npm build on root level.
2. Cd into tegel and run npm run tegel
3. Check in Foundations -> Grid/Spacing/Typography and make sure the stories look good.
4. A couple of random components and make sure the are using the correct px instead of any rem values.

**Screenshots**  
-

**Additional context**  
-
